deployment/log-signer-deployment.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: trillian-log-signer
+  name: trillian-log-signer-deployment
+spec:
+  replicas: 2
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: trillian-log-signer
+    spec:
+      serviceAccountName: trillian-svc
+      restartPolicy: Always
+      containers:
+      - name: trillian-log-signer
+        args: [
+        "$(STORAGE_FLAG)",
+        "--storage_system=$(STORAGE_SYSTEM)",
+        "--etcd_servers=trillian-etcd-cluster-client:2379",
+        "--quota_system=etcd",
+        "--etcd_http_service=trillian-logsigner-http",
+        "--http_endpoint=0.0.0.0:8091",
+        "--sequencer_guard_window=1s",
+        "$(SIGNER_INTERVAL)",
+        "$(SIGNER_NUM_SEQUENCERS)",
+        "$(SIGNER_BATCH_SIZE)",
+        "$(SIGNER_DEQUEUE_BUCKET_FRACTION)",
+        "$(SIGNER_MASTER_HOLD_JITTER)",
+        "--alsologtostderr"
+        ]
+        envFrom:
+        - configMapRef:
+            name: deploy-config
+        image: gcr.io/trillian-opensource-ci/log_signer:v1.3.3
```

Comment:
Ok, I mostly just wanted to know how we'd make it configurable, since we'd need to do so if we wanted to use this for CI deployments. It's fine to defer that for now though.